app/views/motion.erb

```diff
@@ -73,106 +73,49 @@
-                  <span class="icon">
-                      <i class="fa fa-person"></i>
-                    </span>
-                  {{ user.first_name }}
-                </span>
-                
-                <span class="ai-user tag is-medium">
-                  <span class="icon">
-                      <i class="fa fa-person"></i>
-                    </span>
-                  {{ user.first_name }}
-                </span>
-                
-                <span class="ai-user tag is-medium">
-                  <span class="icon">
-                      <i class="fa fa-person"></i>
-                    </span>
-                  {{ user.first_name }}
-                </span>
-      
+                <div class="ai-user"><i class="fa fa-person" aria-hidden="true">{{ user.first_name }}</i></div>
+               <div class="ai-user"><i class="fa fa-person" aria-hidden="true">{{ user.first_name }}</i></div>  
+                <div class="ai-user"><i class="fa fa-person" aria-hidden="true">{{ user.first_name }}</i></div>
+               <div class="ai-user"><i class="fa fa-person" aria-hidden="true">{{ user.first_name }}</i></div>
+               <div class="ai-user"><i class="fa fa-person" aria-hidden="true">{{ user.first_name }}</i></div>
               </div>
             </article>
-            <article class="message">
-              <div class="message-header opposed">
-               <span class="icon">  
-                 <i class="fa fa-close-circle-outline"></i>
-               </span>
+
+            <article class="message opposed">
+              <div class="message-header">
+                <span class="icon">  
+                  <i class="fa fa-close-circle-outline"></i>
+                </span>
                 Opposed
               </div>
               <div class="message-body">
-               <span class="ai-user tag is-medium opposed">
-                 <span class="icon">
-                     <i class="fa fa-person"></i>
-                   </span>
-                 {{ user.first_name }}
-               </span><span class="ai-user tag is-medium opposed">
-                  <span class="icon">
-                      <i class="fa fa-person"></i>
-                    </span>
-                  {{ user.first_name }}
-                </span><span class="ai-user tag is-medium opposed">
-                  <span class="icon">
-                      <i class="fa fa-person"></i>
-                    </span>
-                  {{ user.first_name }}
-                </span>
+                <div class="ai-user"><i class="fa fa-person" aria-hidden="true">{{ user.first_name }}</i></div>
+               <div class="ai-user"><i class="fa fa-person" aria-hidden="true">{{ user.first_name }}</i></div>  
               </div>
             </article>
-            <article class="message">
-              <div class="message-header abstain">
+
+            <article class="message abstain">
```

Comment:
are you adding a new class here? I think the same thing could be achieved using CSS instead. e.g. all this code lives within ai-election, so you could use a 1st child 2nd child 3rd child relationship to just target the one that you want